email-bom/build.gradle

```diff
@@ -1,3 +1,7 @@
 plugins {
     id "io.micronaut.build.internal.bom"
 }
+
+checkVersionCatalogCompatibility {
```

Comment:
@melix is this necessary until we start releasing versions of every module for Micronaut Framework 4?

Reply:
I think it would be better to actually accept the regressions so that they are documented, and potentially shown to users "hey, those are the breaking changes in 4.0". If we disable binary compatibility checks, it's easier for _us_, but we don't do a favor to our users.

Reply:
So the issue was here

https://ge.micronaut.io/s/tec5ayq6zzwfs/console-log?anchor=40&page=1

```
The following libraries were present in the baseline version but missing from this catalog:
 - javax-mail
```

Because `managed-javax-mail` changed to `managed-jakarta-mail` https://github.com/micronaut-projects/micronaut-email/pull/137/files#diff-697f70cdd88ba88fe77eebda60c7e143f6ad1286bca75017421e93ad84fb87df

How do we accept and document this change?

Or do we think we should leave the managed version name the same?

Reply:
Nice, and then we don't have to remind ourselves with an open ticket to revert disabling the check.